ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/Util.hs

```diff
@@ -87,17 +93,26 @@ createSubWidget = do
 
   return (widget, layout)
 
-createCheckBox :: QVBoxLayout.QVBoxLayout -> Text -> IO QCheckBox.QCheckBox
-createCheckBox layout labelText = do
+createCheckBox
+  :: QVBoxLayout.QVBoxLayout
+  -> CheckboxPosition
+  -> Text
+  -> IO QCheckBox.QCheckBox
+createCheckBox layout checkBoxPos labelText = do
   checkBox <- QCheckBox.new
-  QWidget.setSizePolicyRaw checkBox Maximum Maximum
+  QWidget.setSizePolicyRaw checkBox Maximum Preferred
```

Comment:
Can you justify this change? AFAIR, i set `Maximum` so that checkbox never expands *its own label*, i.e. always remains a square.